src/providers/EtherspotUiContextProvider.tsx

```diff
@@ -6,9 +6,10 @@ import EtherspotUiContext from '../contexts/EtherspotUiContext';
 
 // utils
 import { getObjectSortedByKeys, isTestnetChainId, parseEtherspotErrorMessageIfAvailable } from '../utils/common';
+import uniq from 'lodash.uniq';
```

Comment:
can we move this to up to the top lib imports

Reply:
Done 👍